docs: fix minor typo in SDK Bundle description (#74541)
metabase/metabase: docs/embedding/sdk/introduction.md

Starting with Metabase 57, the SDK consists of two parts:

- **SDK Package** – The `@metabase/embedding-sdk-react` npm package is a lightweight bootstrapper library. Its primary purpose is to load and run the main SDK Bundle code.

> - **SDK Bundle** – The full SDK code, served directly from your self-hosted Metabase instance or Metabase Cloud, and it's a part of Metabase. This ensures that the main SDK code is always compatible with its corresponding Metabase instance.

Reviewer: ```suggestion
- **SDK Bundle** – The full SDK code, served directly from your self-hosted Metabase instance or Metabase Cloud. Serving the bundle from your running Metabase ensures that the SDK code is always compatible with your Metabase.
```
